src/app/task.service.ts

```diff
@@ -23,8 +23,14 @@ export class TaskService {
     return of(task);
   }
 
-  getTasks(): Observable<Task[]> {
+  getTasksInProgress(): Observable<Task[]> {
+    this.tasks = this.tasks.filter(t => t.status !== 1);
```

Comment:
This line of code will remove all done tasks from your tasks list.

Reply:
![image](https://user-images.githubusercontent.com/45048289/74611993-50c31200-5101-11ea-90bc-ae89c65e2c58.png)

This way is right? ^^

Reply:
Ok, I think I've found a better way :)